service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -167,6 +173,32 @@ public AsyncJobResult<StartLandingZoneCreation> startLandingZoneCreationJob(
             azureLandingZoneRequest.version()));
   }
 
+  public String startLandingZoneResourceCreationJob(
```

Comment:
Could you add some documentation to this method to explain when you would call it vs. the existing `startLandingZoneCreationJob` method?

Reply:
This works similar to what we have in WSM. This method initiates new flight and return job id.